data/hai/hai reveal 5 philosophers.txt

```diff
@@ -1011,84 +1011,129 @@ mission "Hai Reveal [B13] Freedom"
 			`	"The planet below is a mystery to us. It has a thousand times the mass it realistically should, and its gravitational field reads as such from a great distance. This tremendous gravitational field sustains the large asteroid belt. Despite its incredible mass, as you approach the surface, the field does not increase at anywhere near the rate that it should. Thus, when you stand upon it, the gravity is only similar to that of Earth. There are many forms of radiation and strange materials inside the planet's caves. We believe it is a relic from a forgotten past, but we have no idea what sort of relic it is. Perhaps, now that we are not developing pirate weapons, we can focus on exploration."`
 				goto questions
 			label leave
+			`	After several hours of discussion, during which agent Kopkow prepares the documents for the provisional recognition of the new "Elenctic Commune" with Admiral Danforth and Captain Wallace as witnesses, the wounded are actually attended to in the Elenctic hospital facilities, and most of the human operatives pack up their gear and return to the Auxiliary. Meanwhile Teeneep and Choot'k go to lengths to engage with the Hai here and discover what they would best like to do.`
 			`	There is just one matter which need to be addressed with the three representatives and you raise it with them. "Did we have all of the leaders of this group? Are they all accounted for?"`
 			`	They consult with some of their people who have been scouring the spaceport, helping to account for both the living, and the dead on both sides. At length, just the human comes to you.`
-			`	"Your question was wise, there is one unaccounted for. Before I tell you though I think perhaps you should know more than you do." They reach up and push back their hood, revealing a bald head, as they gesture to the city in general. To your surprise you realize for the first time that they are a woman, though that seems like it could be the least important part of their identity. "Many different overlords we have had, and always they have relied on recruits from the city for some capacities, and have rallied other lesser groups to their banner. As such, ever have their been factions. Some, kinder than others."`
+			`	"Your question was wise, there is one unaccounted for. Before I tell you though I think perhaps you should know more than you do." They reach up and push back their hood, revealing a bald head, as they gesture to the city in general. To your surprise you realize for the first time that they are a woman, though that seems like it could be the least important part of their identity.`
```

Comment:
```suggestion
			`	"Your question was wise, there is one unaccounted for. Before I tell you, though, I think perhaps you should know more than you do." They reach up and push back their hood, revealing a bald head, as they gesture to the city in general. To your surprise you realize for the first time that they are a woman, though that seems like it could be the least important part of their identity.`
```